packages/entity-browser/src/routes/detail/components/DetailView/DetailView.js

```diff
@@ -71,9 +72,9 @@ DetailView.propTypes = {
       React.PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.object])))
   ),
   entity: React.PropTypes.object,
-  loadRelationEntities: React.PropTypes.func,
+  loadRelationEntity: React.PropTypes.func,
```

Comment:
IMO, `loadRelationEntity` and `relationEntities` should be declared as required props, since they're *required* in DetailForm and we simply pass them through, even when they're not defined.